content/docs/source-git/how-to-source-git.md

```diff
@@ -5,13 +5,179 @@ draft: false
 weight: 2
 ---
 
+# How to create a source-git repository?
+
+This guide walks through the steps to create a source-git repository from an
+upstream project.
```

Comment:
Then what's the difference between this section and the already existing `How to create a source-git repository from upstream?` ?